### ISSUE TITLE
<operator-name>.package.yaml was not generated with 0.17.0

### ISSUE THREAD

**Issue by gyliu513** [operator-framework/operator-sdk#2967]
## Bug Report

**What did you do?**
A clear and concise description of the steps you took (or insert a code snippet).

I used the command as follows to generate my CSV file

```
operator-sdk generate csv --make-manifests=false --csv-version 0.0.1 --update-crds=true
```

But this command does not generate `<operator-name>.package.yaml`

```
deploy/
└── olm-catalog
    └── <operator-name>
        ├── 0.0.1
        │   └── <operator-name>.v0.0.1.clusterserviceversion.yaml
        └── <operator-name>.package.yaml << This was lost
```

**What did you expect to see?**
A clear and concise description of what you expected to happen (or insert a code snippet).

**What did you see instead? Under which circumstances?**
A clear and concise description of what you expected to happen (or insert a code snippet).

**Environment**
* operator-sdk version:

```console
root@gyliu-dev1:~/go/src/github.ibm.com/IBMPrivateCloud/ibm-infra-management-install-operator# operator-sdk version
operator-sdk version: "v0.17.0", commit: "2fd7019f856cdb6f6618e2c3c80d15c3c79d1b6c", kubernetes version: "unknown", go version: "go1.13.10 linux/amd64"
```

* go version:

* Kubernetes version information:

* Kubernetes cluster kind: 

* Are you writing your operator in ansible, helm, or go?

**Possible Solution**

**Additional context**

**Comment by gyliu513** [comment]
@camilamacedo86 can you help? Thanks!

**Comment by camilamacedo86** [comment]
Hi @gyliu513,

We removed it from 0.17 but after discussing it better we checked that the package manifest still supported by OLM and then we revert the change. See: https://github.com/operator-framework/operator-sdk/pull/2944. Sorry, for the inconvenient and you can use the master branch until the next release.  

I am closing this as sorted out. However, please feel free to ping us to re-open if you need.

**Comment by gyliu513** [comment]
Thanks @camilamacedo86 , oops, this is a break change, we should at least provide some workaround for this, but not use master branch.

/reopen

**Comment by estroz** [comment]
@gyliu513 I'll be making a patch release that reverts package manifest deprecation fairly soon. We can leave this open until then.

**Comment by gyliu513** [comment]
Thanks @estroz , so you mean you want to release a patch release like 0.17.1? If so, does the new version works with the operator which was generated via 0.17.0?  

For my case, my operator was generated by 0.17.0, and now I want to generate the package bundle, can I continue to use the new 0.17.1 to generate the package bundle?

Look forward to the patch release soon! ;-)

**Comment by estroz** [comment]
@gyliu513 yes v0.17.1 will work with all aspects of a v0.17.0-built project. It is worth bumping patch versions if possible.

**Comment by estroz** [comment]
Fixed in v0.17.1.
